csi/controllerserver.py

```diff
@@ -301,7 +301,7 @@ def DeleteVolume(self, request, context):
 
         delete_volume(request.volume_id)
         logging.info(logf(
-            "Volume deleted",
+            "Volume deleted/archived",
```

Comment:
Add exact message so that from the log it will be easy to identify it is deleted or archived. Also add archived name as `archive_name="archived-" + request.volume_id`